blockchain/types/accountkey/account_key_weighted_multi_sig.go

```diff
@@ -127,7 +151,10 @@ func (a *AccountKeyWeightedMultiSig) AccountCreationGas(currentBlockNumber uint6
 	return numKeys * params.TxAccountCreationGasPerKey, nil
 }
 
-func (a *AccountKeyWeightedMultiSig) SigValidationGas(currentBlockNumber uint64, r RoleType) (uint64, error) {
+func (a *AccountKeyWeightedMultiSig) SigValidationGas(currentBlockNumber uint64, r RoleType, validSigNum int) (uint64, error) {
+	// set isIstanbul
+	isIstanbul := fork.Rules(new(big.Int).SetUint64(currentBlockNumber)).IsIstanbul
```

Comment:
Please initialize variables right before its usage. (to line 168?)
And above comment look unnecessary. (line 156 is what exactly "set isIstanbul" does)

Reply:
Reflected in new change, thanks.